src/transformers/file_utils.py

```diff
@@ -1227,13 +1227,18 @@ def http_user_agent(user_agent: Union[Dict, str, None] = None) -> str:
     """
     Formats a user-agent string with basic info about a request.
     """
+    if DISABLE_TELEMETRY:
+        return "telemetry/off"
```

Comment:
Can we at least keep `transformers/version; telemetry/off`? Our analytics system only parses this if the transformers version comes first.